x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/entity_store/entity_definitions/entity_descriptions/user.ts

```diff
@@ -14,6 +14,14 @@ export const userEntityEngineDescription: EntityDescription = {
   entityType: 'user',
   version: USER_DEFINITION_VERSION,
   identityField: USER_IDENTITY_FIELD,
+  identityFieldMapping: {
+    type: 'keyword',
+    fields: {
+      text: {
+        type: 'match_only_text',
+      },
+    },
+  },
```

Comment:
We are keeping `user.name` as a multi-field because of ECS [instructions](https://www.elastic.co/docs/reference/ecs/ecs-user#field-user-name), right?

Reply:
yes